src/parser/jobs/whm/modules/Thinair.tsx

```diff
@@ -18,6 +18,12 @@ interface ThinAirRecord {
 	mpsaved: number,
 }
 
+const USAGE = {
+	BAD: 'red',
+	OKAY: 'yellow',
+	GOOD: 'green',
```

Comment:
I would change these to some hex colors. I'm not sure what our standard colors are, but ackwell can probably tell you where to look.

Reply:
Checklist uses the named colours so this is likely fine.